tests/network/migration/test_masquerade_connectivity_after_migration.py

```diff
@@ -20,6 +20,21 @@
 LOGGER = logging.getLogger(__name__)
 
 
+def ping_via_console(src_vm, dst_vm):
+    """
+    Ping from src_vm to dst_vm via VM console to avoid SSH masking issues.
```

Comment:
please use proper google dcosting formatting

Reply:
Thanks for the feedback. I agree consistency helps. I updated this function to a proper google-style docstring.  +1 on adding a linter rule to help enforcing consistency across the repo.